release-notes/index.md

```diff
@@ -4,15 +4,10 @@ keywords: docker, documentation, about, technology, understanding,  release
 title: Docker release notes
 ---
 
-Find out what's new in Docker! Release notes also contain detailed information about known issues and fixes for each component. 
+Find out what's new in Docker! Release notes can be found in the product manuals:
 
-- [Docker Engine](/engine/release-notes)
-- [Docker Desktop Enterprise](/ee/desktop/release-notes/)
-- [Docker Desktop for Mac](/docker-for-mac/release-notes.md) 
-- [Docker Desktop for Mac Edge Releases](/docker-for-mac/edge-release-notes.md)
-- [Docker Desktop for Windows](/docker-for-windows/release-notes.md) 
-- [Docker Desktop for Windows Edge Releases](/docker-for-windows/edge-release-notes.md)
-- [Docker for AWS](/docker-for-aws/release-notes.md)
-- [Docker Compose](docker-compose.md)
-- [Docker Machine](docker-machine.md)
-- [Docker Swarm (standalone)](docker-swarm.md)
+- [Docker Engine](/engine/release-notes/)
+- [Docker Desktop for Mac](/docker-for-mac/) 
+- [Docker Desktop for Windows](/docker-for-windows/) 
+- [Docker Desktop Enterprise](/ee/desktop/)
+- [Docker Compose](/compose/)
```

Comment:
Can we add Hub release notes to this list [Docker Hub](/docker-hub/release-notes/)?

Reply:
Oh, yes! Didn't realise we had a changelog for that now; I'll add

I'm a bit on the fence still how prominent we want this page to be (perhaps even remove it from the navigation); we _must_ keep the page itself though, because `/release-notes/` is a likely URL for people to discover, and it should guide them to the relevant pages.

Perhaps with the rewrite of the landing-page we can use the landing page to help discover "what's new" in any of the products

Reply:
Yea, agree with you on retaining the page. Considering that we are planning to surface a 'What's new' section in the new landing page, we can use this page to navigate users to learn about new features across all components.